### PR TITLE
remove ownership cleanliness submission toggle and related code/tests

### PR DESCRIPTION
**Pull request by czue**
@snopoke @dannyroberts 

this is just the flag that starts tracking it on submission. new restore is still only enabled for a small subset of domains.

### CODE REVIEW

**Comment by czue** [comment]
(wait for tests)

File: corehq/ex-submodules/casexml/apps/phone/cleanliness.py [hunk 1]
```diff
@@ -92,15 +84,9 @@ def set_cleanliness_flags(domain, owner_id, force_full=False):
...
-            # on submission
-            cleanliness_obj.is_clean and not OWNERSHIP_CLEANLINESS.enabled(domain)
-        ) or (
-            # if dirty, first check the hint and only do a full check if it's not valid
-            not cleanliness_object.is_clean and (
-                not cleanliness_object.hint or not hint_still_valid(domain, owner_id, cleanliness_object.hint)
-            )
+        # if dirty, first check the hint and only do a full check if it's not valid
+        return not cleanliness_object.is_clean and (
+           not cleanliness_object.hint or not hint_still_valid(domain, owner_id, cleanliness_object.hint)
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/cleanliness.py]: E121 continuation line under-indented for hanging indent

**Comment by czue** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
Exciting!

**Pull request merged**